rocky/rocky/views/ooi_mute.py

```diff
@@ -24,3 +29,17 @@ def get_context_data(self, **kwargs):
         context["ooi_type"] = MutedFinding.get_object_type()
 
         return context
+
+
+class MuteFindingsBulkView(SingleOOIMixin):
+    def post(self, request, *args, **kwargs):
+        self.ooi_class = MutedFinding
```

Comment:
This can be a class property:
```suggestion
    ooi_class = MutedFinding

    def post(self, request, *args, **kwargs):
```